app/src/main/java/org/schulcloud/mobile/controllers/dashboard/NewsWidget.kt

```diff
@@ -0,0 +1,59 @@
+package org.schulcloud.mobile.controllers.dashboard
+
+import android.arch.lifecycle.Observer
+import android.arch.lifecycle.ViewModelProviders
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import kotlinx.android.synthetic.main.widget_news.*
+import org.schulcloud.mobile.R
+import org.schulcloud.mobile.controllers.main.NewsListAdapter
+import org.schulcloud.mobile.controllers.main.NewsListFragment
+import org.schulcloud.mobile.models.news.NewsRepository
+import org.schulcloud.mobile.viewmodels.NewsListViewModel
+import org.schulcloud.mobile.views.MiddleDividerItemDecoration
+import org.schulcloud.mobile.views.NoScrollLinearLayoutManager
+
+class NewsWidget : Widget() {
+    companion object {
+        val TAG: String = NewsWidget::class.java.simpleName
+    }
+
+    private lateinit var viewModel: NewsListViewModel
+    private val newsAdapter: NewsListAdapter by lazy {
+        NewsListAdapter()
```

Comment:
Because we don't have a NewsActivity yet we decided to show everything for now and deal with cleaning up the widget as well as NewsListFragment in a later PR.